src/support/ios.c

```diff
@@ -1107,6 +1109,10 @@ int ios_getutf8(ios_t *s, uint32_t *pwc)
     c0 = (char)c;
     if ((unsigned char)c0 < 0x80) {
         *pwc = (uint32_t)(unsigned char)c0;
+        if (c == '\n')
+            s->u_colno = 0;
```

Comment:
`= 1`?

Reply:
No.  It only becomes `colno = 1` when it reads the first char.

Reply:
(I think this was an error in #9579.)